pkg/controller/jobs/statefulset/statefulset_webhook.go

```diff
@@ -140,6 +139,6 @@ func (wh *Webhook) ValidateDelete(context.Context, runtime.Object) (warnings adm
 	return nil, nil
 }
 
-func GetWorkloadName(statefulSetName string, statefulSetUID types.UID) string {
-	return jobframework.GetWorkloadNameForOwnerWithGVK(statefulSetName, statefulSetUID, gvk)
+func GetWorkloadName(statefulSetName string) string {
+	return jobframework.GetWorkloadNameForOwnerWithGVK(statefulSetName, "", gvk)
```

Comment:
Please add a comment: "Passing empty UID as it is not available before object creation"

Reply:
Good point! Thanks. Done.